modules/nf-core/nonpareil/curve/meta.yml

```diff
@@ -0,0 +1,48 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/yaml-schema.json
+name: "nonpareil_curve"
+description: Visualise metagenome redundancy curve from a single Nonpareil npo file
+keywords:
+  - metagenomics
+  - statistics
+  - coverage
+  - complexity
+  - redundancy
+  - diversity
+  - visualisation
+tools:
+  - "nonpareil":
+      description: "Estimate average coverage and create curves for metagenomic datasets"
+      homepage: "https://github.com/lmrodriguezr/nonpareil"
+      documentation: "https://nonpareil.readthedocs.io/en/latest/"
+      tool_dev_url: "https://github.com/lmrodriguezr/nonpareil"
+      doi: "10.1128/msystems.00039-"
+      licence: ["Artistic License 2.0"]
```

Comment:
I think the [] are not needed, or won't be soon thanks to @mahesh-panchal

Reply:
OK will try